### PR TITLE
Added shell test for `cfbs clean`

### PR DESCRIPTION
**Pull request by larsewi**
Ticket: None
Changelog: None
Signed-off-by: Lars Erik Wik <lars.erik.wik@northern.tech>

### CODE REVIEW

File: test/shell/009_clean.sh [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+mkdir -p ./tmp/
+cd ./tmp/
+touch cfbs.json && rm cfbs.json
+
+cfbs init
+cfbs add promise-type-git
+grep '"name": "library-for-promise-types-in-python"' cfbs.json
+grep '"name": "promise-type-git"' cfbs.json
+cfbs remove promise-type-git
+cfbs clean
```
Comment (olehermanse) [inline test/shell/009_clean.sh:13]: I guess you should use `--non-interactive` here :)

File: test/shell/009_clean.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+set -e
```
Comment (olehermanse) [inline test/shell/009_clean.sh:1]: Please add this to the `all.sh` script in the same folder.

**Review by olehermanse — CHANGES_REQUESTED** [review]

**Review by craigcomstock — COMMENTED** [review]
Agree with @olehermanse comments. Otherwise looks good.

**Review by olehermanse — CHANGES_REQUESTED** [review]
Please rebase this now that the other one was merged. In hindsight it probably would have been better to do 2 commits in 1 PR in this case :)

**Comment by olehermanse** [comment]
> Please rebase this now that the other one was merged. In hindsight it probably would have been better to do 2 commits in 1 PR in this case :)

Never mind, I fixed the conflict in GitHub UI :) Merging.

**Pull request merged**